pkg/apis/camel/v1/integration_types.go

```diff
@@ -262,6 +262,8 @@ type IntegrationCondition struct {
 	Reason string `json:"reason,omitempty"`
 	// A human-readable message indicating details about the transition.
 	Message string `json:"message,omitempty"`
+	// TODO: add documentation
```

Comment:
Nitpick... you know otherwise nobody will ever bother to fix it :)

Reply:
done